content/en/cli/getting-started.md

```diff
@@ -70,7 +70,8 @@ preact build --json
 ## Editing index.html
 
 If you want to add or edit the markup generated by `preact-cli` to add meta tags, custom scripts or fonts, then you can edit `src/template.html`:
-This is generated by `preact-cli` v3.
+This is generated by `preact-cli` v3.  
+`src/template.html` is rendered by [EJS](https://ejs.co/).
```

Comment:
This is generated by `preact-cli` v3 and is rendered using [EJS](https://ejs.co/).

Reply:
Thank you for reviewing.
I changed it.